src/main/java/mate/jdbc/service/ManufacturerServiceImpl.java

```diff
@@ -0,0 +1,38 @@
+package mate.jdbc.service;
+
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.model.Manufacturer;
+
+public class ManufacturerServiceImpl implements ManufacturerService {
+
+    @Inject
+    private ManufacturerDao manufacturerDao;
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        return manufacturerDao.create(manufacturer);
+    }
+
+    @Override
+    public Optional<Manufacturer> get(Long id) {
+        return manufacturerDao.get(id);
+    }
```

Comment:
checklist 
find() methods in Dao layer should return Optional (not in services). In the service layer better call method get() on Optional and return the object.

 lets rewrite this method and also catch Exception.